### PR TITLE
Run filter from CLI

### PR DESCRIPTION
**Pull request by simoncozens**
This allows the user to run a ufo2ft filter (or filters) on a UFO file and save it from the command line. This is helpful for testing and debugging filters, as well as doing general filter-based operations on fonts.

```
$ python3 -m ufo2ft.filters ../glyphsLib/master_ufo/Nunito-M1000.ufo glyphsLib.filters.eraseOpenCorners
INFO:ufo2ft.filters:Running EraseOpenCornersFilter on Nunito-M1000
INFO:ufo2ft.filters:Written on ../glyphsLib/master_ufo/Nunito-M1000#1.ufo
```

### CODE REVIEW

File: Lib/ufo2ft/filters/__main__.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "--exclude", metavar="GLYPHS", help="comma-separated list of glyphs to not filter"
+)
+parser.add_argument("ufo", metavar="UFO", help="UFO file")
+parser.add_argument("filters", metavar="FILTER", nargs="+", help="filter name")
+
+args = parser.parse_args()
+if not args.output:
+    args.output = makeOutputFileName(args.ufo)
+
+ufo = defcon.Font(args.ufo)
```
Comment (anthrotype) [inline Lib/ufo2ft/filters/__main__.py:27]: do you mind switching to ufoLib2 instead? or try ufoLib2 then defcon then ImportError

**Review by anthrotype — APPROVED** [review]
This is very useful, thanks!

**Comment by moyogo** [comment]
You can use loadFilterFromString() now that #466 has been merged.

**Comment by anthrotype** [comment]
> You can use loadFilterFromString() now that #466 has been merged.

good point, yes please

**Comment by moyogo** [comment]
LGTM

**Comment by anthrotype** [comment]
I modified this slightly to set the include/exclude attribute on the filter instance _after_ it has been initialized from the CLI string, this way we can support initializing the filter not only using the filter class name but also with optional keyword arguments

```
$ python -m ufo2ft.filters tests/data/TestFont.ufo 'CubicToQuadraticFilter(reverseDirection=False)'
```

https://github.com/googlefonts/ufo2ft/commit/d15836da0b6aeae7e3d1e770cd839b86230b4d98

**Pull request merged**